tools/common/Frameworks.cs

```diff
@@ -305,6 +306,7 @@ public static Frameworks GetiOSFrameworks (Application app)
 				{ "IdentityLookupUI", "IdentityLookupUI", 12,0 },
 				{ "NaturalLanguage", "NaturalLanguage", 12,0 },
 				{ "Network", "Network", 12, 0 },
+				{ "SoundAnalysis", "SoundAnalysis", 13, 0 },
```

Comment:
If you look at the lines above, you'll see that there's an empty line between entries for each new OS version (this goes for watchOS and tvOS frameworks below as well).